webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue

```diff
@@ -198,6 +198,9 @@ export default {
     authorAvatarURL() {
       return this.news && (this.news.profileAvatarURL || this.news.authorAvatarUrl);
     },
+    backURL() {
+      return this.news && this.news.url.includes('/news/detail?content-id=') ? `${eXo.env.portal.context}/${eXo.env.portal.portalName}` : this.news.spaceUrl;
```

Comment:
I prefer to check correctly if the user is a member of the news space. We can not base on urls check because it can change and in that case we should return here again to follow.